tests/all-tests.js

```diff
@@ -0,0 +1,159 @@
+/* Tests for the "pouch" module */
+"use strict";
+var fluid      = fluid || require("infusion");
+var gpii       = fluid.registerNamespace("gpii");
+var path       = require("path");
+var jqUnit     = fluid.require("jqUnit");
+var request    = require("request");
+
+require("gpii-express");
+
+require("../src/js/pouch.js");
+
+function isSaneResponse(jqUnit, error, response, body, status) {
```

Comment:
These should all go in a gpii.tests.pouchdb namespace. You should consider consolidating all of your "isSaneResponse" functions within gpii-express itself.